image.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="image.css">
+</head>
+
+<body>
+    <div class="imag1">
+        <a href="index.html"><img src="https://images-eu.ssl-images-amazon.com/images/I/510OKt7Z1CS._SX300_SY300_QL70_FMwebp_.jpg" alt="Developers Page" width=100 height=100></a>
+        <p>
+            "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam,
```

Comment:
align the camera picture to right , and change the paragraph language cause when page is converting to english its changing the text right to left in the lists.  rest everything is good